ait/core/server/handler.py

```diff
@@ -0,0 +1,106 @@
+import __builtin__
+
+from ait.core import log, tlm
+
+
+class Handler(object):
+    """
+    This is the base Handler class that all custom handlers must inherit
+    from. All custom handlers must implement the handle method, which will
+    called by the stream the handler is attached to when the stream receives
+    data.
+    """
+
+    def __init__(self, input_type=None, output_type=None, **kwargs):
+        self.input_type = input_type
+        self.output_type = output_type
+
+        for key, value in kwargs.items():
+            setattr(self, key, value)
+
+    def __repr__(self):
+        return '<handler.%s>' % (self.__class__.__name__)
+
+    def validate_input(self, input_data):
```

Comment:
I'm still uncertain of whether we need to care about validating stuff. Could you elaborate on why you think it's a good choice? I'm probably overlooking something and would like to hear other perspectives.

Reply:
I agree this is unnecessary. Will remove.